extensions/cornerstone-dicom-sr/package.json

```diff
@@ -46,7 +46,7 @@
     "@babel/runtime": "^7.20.13",
     "classnames": "^2.3.2",
     "@cornerstonejs/adapters": "^0.4.1",
-    "@cornerstonejs/core": "^0.36.5",
-    "@cornerstonejs/tools": "^0.55.1"
+    "@cornerstonejs/core": "^0.37.0",
+    "@cornerstonejs/tools": "^0.57.1"
```

Comment:
Why not latest - I think that is 0.38.0, 0.58.0 and adapters 0.6.0
Also, please ensure you update every package.json correctly - getting mismatches is ugly for including the libraries.